appveyor.yml

```diff
@@ -1,11 +1,65 @@
 image: Visual Studio 2015
 
+environment:
+  NUGET_API_KEY:
+    secure: Rh/UHYn7HGj8a7LFWo9C1rSA0sHSQNV3wIqj9bvwRR8yLmCXu9JmbNkmkctk2cBn
```

Comment:
I haven't been able to keep up with the issues here due to lack of time. I'll still be watching, and I'll try to "chime in" where I can, but I'm not going to have much time available in the coming weeks 😢